lib/reporters/cli/index.js

```diff
@@ -233,9 +236,15 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
                 `${mime.mimeType}`,
                 `${mime.mimeFormat}`,
                 `${mime.charset}`
-            ].join(` ${colors.gray(symbols.star)} `);
-
+            ].join(` ${colors.gray(symbols.star)} `),
+            // eslint-disable-next-line max-len
+            bodyClipSize = reporterOptions.verboseOutputLimit ? bytes(reporterOptions.verboseOutputLimit) : bytes(DEFAULT_VERBOSE_OUTPUT_TRUNCATION_LIMIT);
```

Comment:
Set the default value here. No need to compute. Instead at the below `if` block, set it if `!isNaN` and warn user in else of that block